src/core/PWSfileV4.h

```diff
@@ -53,7 +53,7 @@ class PWSfileV4 : public PWSfile
                      unsigned char *&content, size_t clen);
 
   uint32 GetNHashIters() const {return m_nHashIters;}
-  void SetNHashIters(uint32 N) {m_nHashIters = N;}
+  void SetNHashIters(uint32 N) {m_nHashIters = N / HASH_FACTOR;}
```

Comment:
Is it intended that Get after Set does not return the original N?

Reply:
Fixed - thanks!